.github/workflows/automatic_pull_request_review.yml

```diff
@@ -48,5 +34,8 @@ jobs:
       env:
         CI: true
 
+    - name: Lint
+      run: make lint
+
```

Comment:
@minhnimble Thanks for noticing this!

Actually, if we just run `swiftlint` (warning without fail) the order of `Lint` and `Build` jobs doesn't matter. But since we run it with `--strict` parameter, it's better to put the `Lint` job first so we don't waste time to build if there's linting errors 🙏 

Will update soon 👍